src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculable, Printable {

```

Comment:
According to the checklist, you should not use abstract classes to set behavior for classes. Consider using an interface or a concrete class instead, depending on the use case.